sys-boot/grub/grub-2.12-r2.ebuild

```diff
@@ -315,6 +315,9 @@ src_install() {
 	# https://bugs.gentoo.org/231935
 	dostrip -x /usr/lib/grub
 
+	sed -e "s/%PV%/${PV}/" < "${FILESDIR}/sbat.csv" > "${ED}/usr/share/grub/sbat.csv" \
```

Comment:
I would rather sed to a temporary file and then install using doins.

Also, please pass the input file as an argument to sed instead of using shell redirection to connect it to stdin.

Reply:
Better like this?